linchpin/__init__.py

```diff
@@ -383,17 +394,86 @@ def _validate_topology(self, topology):
         return resources
 
 
-    def do_action(self, provision_data, action='up', run_id=None):
+    def get_pf_data_from_rundb(self, targets, run_id=None, tx_id=None):
+        """
+        This function takes the action and provision_data, returns the
+        pinfile data
+
+        :param targets: A list of targets for which to get the data
+
+        :param action: Action taken (up, destroy, etc). (Default: up)
```

Comment:
I don't see param action anymore...  and it looks to be hard coded to 'up' now.

Reply:
Are you referring to the do_action method? It got moved further down in the code is all.

https://github.com/CentOS-PaaS-SIG/linchpin/pull/512/files/8719bd6245db8a6fa5c6e0df908f65a4a3605e02#diff-eda7cdccb38e3e91756c08ffdbc7ac4e

Line 463 from what I see...

Reply:
Please update the inline docs for the function then.  Thanks.